src/tigerbeetle/cli.zig

```diff
@@ -158,6 +158,14 @@ const CliArgs = union(enum) {
     , .{
         .default_address = constants.address,
         .default_port = constants.port,
+        .default_cache_grid = @divExact(
+            constants.grid_cache_size_default,
+            1024 * 1024 * 1024,
+        ),
+        .default_memory_lsm_manifest = @divExact(
```

Comment:
nit, but maybe use `default_cache_grid_gb` and `default_memory_lsm_manifest_mb` to connect the calculation and the actual help message?

Reply:
Done.